packages/@okta/vuepress-site/guides/sign-into-your-web-app/add-signin-button/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Add a Button to Sign In
+---
+In your application, you want to provide a button to sign the user in. When the user clicks this button, they are redirected to the Okta-hosted sign-in page so they can authenticate.
```

Comment:
I'd remove that comma: `In your application you want to`

Reply:
done